doc/architecture/decisions/0002-object-identifiers.md

```diff
@@ -8,32 +8,38 @@ Agreed
 
 ## Context
 
-To uniquely identify an object in Ensembl we need three pieces of information:
+To uniquely identify an entity in Ensembl (e.g. gene, transcript, region/slice/location) we need three pieces of information:
 
 -   The **dataset** to which the object belongs, e.g. homo_sapiens_GCA_000001405_14 [^1]
 -   The **object type**, e.g. gene, transcript, region  
 -   The **object ID**, e.g. ENSG00000139618, AT3G52430, 13:32315474-32400266 [^2]  
   
-Ensembl objects are used extensively in the web client and passing around three separate pieces of information for each one can be cumbersome, error prone, and adds additional work when serialisation is required (e.g. for use in URLs or hash keys).  
+Ensembl entities (called objects) are used extensively in the web client and passing around three separate pieces of information for each one can be cumbersome, error prone, and adds additional work when serialisation is required (e.g. for use in URLs or hash keys). Therefore creating a single string which can be decomposed to the 3 data tuple is useful.
```

Comment:
> Ensembl entities (called objects) 
We are actually thinking of renaming them to features 😄

Reply:
We need to document the now (with a point to the future). Would this prevent the ADR being accepted?